gulp/browserSync.js

```diff
@@ -4,6 +4,8 @@
 // External dependencies
 import requireUncached from 'require-uncached';
 import browserSync from 'browser-sync';
+import log from 'fancy-log';
```

Comment:
Hey @ataylorme! I don't mind introducing logging and colors, but let's do so in its own PR. Then we can add it for other functionality.

Reply:
The logging packages are already in (see [https://github.com/wprig/wprig/pull/47#discussion_r201160630](https://github.com/wprig/wprig/pull/47#discussion_r201160630)), I am just making use of them. 

Would you prefer using log messages to be a separate PR later?

Reply:
I'm fine with keeping the logging in. Thanks!